src/server/ghService/import/importPackageUtil.ts

```diff
@@ -5,41 +5,61 @@ import fetch from "node-fetch";
 import * as jsYaml from "js-yaml";
 import { ManifestFolderList } from "../types/import/manifestFolderListModel";
 
-
-const {
-  GITHUB_TOKEN,
-} = process.env;
+const { GITHUB_TOKEN } = process.env;
 
 const CONTENTS_BASE_URL = "https://api.github.com/repos/microsoft/winget-pkgs/contents";
 
-//! only call for initial import
-const getManifestFolderPaths = async (): Promise<string[]> => {
-  const manifestFolderList: Promise<ManifestFolderList[]> = await fetch(
-    `${CONTENTS_BASE_URL}/manifests`, {
+// TODO actually check it works
+const getMPaths = async (): Promise<string[]> => {
+  const mFolderPaths: Promise<ManifestFolderList[]> = await fetch(
+    `${CONTENTS_BASE_URL}/manifests`,
+    {
       headers: {
         Authorization: `token ${GITHUB_TOKEN}`,
       },
     },
   ).then((res) => res.json());
 
+  const mpaths = (await mFolderPaths).map((x) => x.path);
+
+  return mpaths;
+};
+
+// TODO actually check it works
+const getManifestFolderPaths = async (): Promise<string[]> => {
+  const mPaths = await getMPaths();
+  const manifestFolderList: ManifestFolderList[] = [];
+
+  for (let i = 0; i < mPaths.length; i += 1) {
```

Comment:
you can do for (let whateverYouWant of mPaths) here

Reply:
doing that did cause a problem if I remember correctly, also shooo